examples/run_Hahn_echo.py

```diff
@@ -92,7 +92,9 @@
 )
 data.setaxis("ph1", ph1_cyc / 4)
 data.reorder(["ph1", "nScans", "t2"])
-data.set_prop("postproc_type", "spincore_SE_v1")
+data.set_prop("postproc_type", "spincore_generalproc_v1")
```

Comment:
You can't say that the post processing type is this. This is the name of a post-processing function that is a general function that is used by many different post-processing types